server/tests/api/server/stats.ts

```diff
@@ -121,6 +124,10 @@ describe('Test stats (excluding redundancy)', function () {
       expect(data.totalDailyActiveUsers).to.equal(1)
       expect(data.totalWeeklyActiveUsers).to.equal(1)
       expect(data.totalMonthlyActiveUsers).to.equal(1)
+
+      expect(data.totalLocalDailyActiveVideoChannels).to.equal(1)
```

Comment:
Only count that active channels === 1 is not enough: create another channel and check it's taken into account after an upload and not before

Also please correctly test `totalLocalPlaylists`